### PR TITLE
Css grid

### PR DESCRIPTION
**Pull request by felipesere**
@RabeaGleissner @fabientownsend

### CODE REVIEW

File: assets/css/main.scss [hunk 1]
```diff
@@ -212,6 +212,39 @@ hr.separator {
...
 // ==== PEOPLE PICKER ==== //
 
+.people-grid {
+  display: grid;
+  grid-auto-rows: 1fr;
+  grid-gap: 20px;
+  grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+}
+
+.advisor-grid-tile {
```
Comment (fabientownsend) [inline assets/css/main.scss]: Does the following class need to be nested in `.advisor-grid-tile`?

Comment (felipesere) [reply assets/css/main.scss]: I _think_ not... it should be specific enough, right?

Comment (fabientownsend) [reply assets/css/main.scss]: Yep!

File: assets/backstop_data/engine_scripts/casper/clickAndHoverHelper.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+var WAIT_TIMEOUT = 5000;
+
+module.exports = function(casper, scenario) {
+  var waitFor = require('./waitForHelperHelper')(casper, WAIT_TIMEOUT);
```
Comment (fabientownsend) [inline assets/backstop_data/engine_scripts/casper/clickAndHoverHelper.js]: I think you can replace your var by let/const

File: assets/backstop_data/engine_scripts/casper/loadCookies.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+var fs = require('fs');
+
+module.exports = function (casper, scenario) {
+  var cookies = [];
+  var cookiePath = scenario.cookiePath;
+
+  // READ COOKIES FROM FILE IF EXISTS
```
Comment (fabientownsend) [inline assets/backstop_data/engine_scripts/casper/loadCookies.js]: I'm not sure this comment is require

File: assets/backstop_data/engine_scripts/casper/loadCookies.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+var fs = require('fs');
+
+module.exports = function (casper, scenario) {
+  var cookies = [];
```
Comment (fabientownsend) [inline assets/backstop_data/engine_scripts/casper/loadCookies.js]: I'm not quite sure that this variable require to be initialized with an empty array

File: assets/backstop_data/engine_scripts/casper/clickAndHoverHelper.js (hunk 2 above)
Comment (felipesere) [reply assets/backstop_data/engine_scripts/casper/clickAndHoverHelper.js]: Oh all of these are not mine 😸

File: assets/backstop_data/engine_scripts/casper/loadCookies.js (hunk 4 above)
Comment (felipesere) [reply assets/backstop_data/engine_scripts/casper/loadCookies.js]: Oh all of these are not mine 😸

File: assets/backstop_data/engine_scripts/casper/loadCookies.js (hunk 3 above)
Comment (felipesere) [reply assets/backstop_data/engine_scripts/casper/loadCookies.js]: Oh all of these are not mine 😸

File: assets/backstop_data/engine_scripts/casper/clickAndHoverHelper.js (hunk 2 above)
Comment (RabeaGleissner) [reply assets/backstop_data/engine_scripts/casper/clickAndHoverHelper.js]: Maybe you could put this in a `vendor/` directory. I think that's the convention for external libraries.

File: assets/backstop_data/engine_scripts/casper/loadCookies.js (hunk 3 above)
Comment (RabeaGleissner) [reply assets/backstop_data/engine_scripts/casper/loadCookies.js]: @fabientownsend maybe you should raise a PR on the BackstopJS repo 😂

File: assets/css/main.scss [hunk 5]
```diff
@@ -212,47 +212,42 @@ hr.separator {
 
 // ==== PEOPLE PICKER ==== //
 
-.people-picker {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
+.people-grid {
+  display: grid;
```
Comment (RabeaGleissner) [inline assets/css/main.scss]: So modern and progressive! Such wow 😮  👍

**Pull request merged**